app/components/rails_bootstrap/badge_component.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+class RailsBootstrap::BadgeComponent < ApplicationComponent
+  # rename variant to variation 
+  # variant is a keyword in rails
```

Comment:
Can we remove this comment, we don't really need them
Few reasons why: 
- Code should be self-explanatory and Ruby is known for its human redable syntax
- We are not completely sure if variant is a keyword or not. That was just an assumption since things were not working when using `variant`. So let's not write something we are not sure about

Reply:
comments are removed.